__tests__/testHelpers.js

```diff
@@ -27,3 +28,18 @@ export const mockPromiseRejected = (message = 'test error') =>
       reject(new Error(message))
     })
   })
+
+// polls for an enzyme element existence for 2s
+export const waitForElement = async (
+  wrapper,
+  selector,
+  config = { attempts: 10, frequency: 200 }
```

Comment:
On line 32 you stated `frequency` should be `2s`, however its at `200ms`

Reply:
10 attempts x 200ms frequency = 2 seconds 😁